workflows/toast_so_sim.py

```diff
@@ -116,6 +116,22 @@ def parse_config(operators, templates, comm):
         help="The output directory",
     )
 
+    parser.add_argument(
```

Comment:
This looks unfinished. We already supported `--save_hdf5.enable`. You also didn't add saveBooks to the list of operators.

Reply:
I added support back to the script (I had intended to remove support).  I strongly advise saving the output from one or more simulation runs as HDF5 first, and then run the `toast_so_convert.py` script on the results to generate books with wafer data containing all frequencies.  If you run a single-frequency simulation and save to books, half the timestreams will be zero.